src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.EqualableValueTC.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal static partial class ValueSetFactory
+    {
+        /// <summary>
+        /// A type class for values that can be directly compared using equality.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        private interface EqualableValueTC<T>
```

Comment:
>EqualableValueTC [](start = 26, length = 16)

`IEqualableValueTC`?

Reply:
The convention for type classes is different than for other interfaces (`NameTC`)

---
In reply to: [381025631](https://github.com/dotnet/roslyn/pull/41503#discussion_r381025631) [](ancestors = 381025631)